src/doc/trpl/match.md

```diff
@@ -20,25 +20,26 @@ match x {
 }
 ```
 
-`match` takes an expression and then branches based on its value. Each *arm* of
-the branch is of the form `val => expression`. When the value matches, that arm's
+`match` takes an expression and then branches based on its value. Each branch or
+*case* is of the form `val => expression`. When the value matches, that case's
```

Comment:
SML, Haskell, and Erlang use a `case` keyword in pattern matching, in Scala `case` delineates the clauses in a `match` expression; these are a few that I had in mind, along with formal relationship to [case analysis](http://en.wikipedia.org/wiki/Case_analysis#Exhaustive_analysis).

Anyhow, if "arm" has become pervasive in Rust corpus, I'll drop this isolated commit when rebasing.